libr/core/cmd_debug.c

```diff
@@ -2671,7 +2700,7 @@ static int cmd_debug_continue (RCore *core, const char *input) {
 	switch (input[1]) {
 		case 0: // "dc"
 			r_reg_arena_swap (core->dbg->reg, true);
-			r_debug_continue (core->dbg);
+			r_debug_continue (core->dbg, core);
```

Comment:
it is a bad idea to change the signature of all the debug functions to accept one more argument.

also, passing core to debug is inconsistent because core depends on debug.